### ISSUE TITLE
error after run "jrnl"-command

### ISSUE THREAD

**Issue by velasco** [jrnl-org/jrnl#718]
Hi,

I managed to install it as well as the encryption but once I run the jrnl command, I get the following error:

```
Traceback (most recent call last):
  File "/usr/local/bin/jrnl", line 8, in <module>
    sys.exit(run())
  File "/usr/local/lib/python2.7/dist-packages/jrnl/cli.py", line 194, in run
    journal = Journal.Journal(journal_name, **config)
  File "/usr/local/lib/python2.7/dist-packages/jrnl/Journal.py", line 39, in __init__
    self.open()
  File "/usr/local/lib/python2.7/dist-packages/jrnl/Journal.py", line 105, in open
    journal = util.get_password(keychain=self.name, validator=validate_password)
  File "/usr/local/lib/python2.7/dist-packages/jrnl/util.py", line 37, in get_password
    pwd_from_keychain = keychain and get_keychain(keychain)
  File "/usr/local/lib/python2.7/dist-packages/jrnl/util.py", line 56, in get_keychain
    return keyring.get_password('jrnl', journal_name)
  File "/usr/local/lib/python2.7/dist-packages/keyring/core.py", line 57, in get_password
    return _keyring_backend.get_password(service_name, username)
  File "/usr/local/lib/python2.7/dist-packages/keyring/backends/fail.py", line 24, in get_password
    raise RuntimeError(msg)
RuntimeError: No recommended backend was available. Install a recommended 3rd party backend package; or, install the keyrings.alt package if you want to use the non-recommended backends. See https://pypi.org/project/keyring for details.
```
Any ideas what could help?

Regards,

**Comment by maebert** [comment]
Hey @velasco, which OS are you using?

If you're on [Ubuntu](https://pypi.org/project/keyring/#using-keyring-on-ubuntu-16-04), try running `pip install secretstorage dbus-python`

**Comment by peter-schmidbauer** [comment]
This should get fixed by #699.

**Comment by maebert** [comment]
Closed as duplicate of #654
